lassen/utils.py

```diff
@@ -0,0 +1,57 @@
+from codecs import decode
+import struct
+
```

Comment:
We should be using hwtypes.FPVector in order to construct floating point values and for converting between FP and BitVector.